### ISSUE TITLE
Clean things up

### ISSUE THREAD

**Issue by deepak-ramachandra** [deepakhr1999/NamedEntityRecognition#1]
A lot of code, especially in a notebook format. Add more documentation so that someone visiting is able to make use of this code. The aim is to be _instructive and illustrative_.

Support this for a linux machine with GPUs. There should be two environments - 
1. For training, with pytorch with cuda installation.
2. For inference with the webapp, with simple lightweight packages.

After documenting the installation instructions
1. Fix bugs, if any.
2. Organize code into `.py` files. Parse arguments from command-line.
3. Add a pylint file along which you maintain the code quality. Perhaps checkout something from [google](https://google.github.io/styleguide/pyguide.html).

Readme file
1. Talk a bit about the paper you implemented. Credit the authors and cite anything you can.
2. Installation instructions
3. High level explanation of the code
4. What data you are using.
5. You've spent a lot of time writing beam-search, perhaps write about it?
6. Show some training plots - that the training loss is decreasing, f1 is improving over iterations.
7. Add pictures if possible.
